src/pages/generate/constants/curationDetectionMode.ts

```
@@ -0,0 +1,22 @@
+export type CurationDetectionMode = 'server' | 'client';
+
+export const CURATION_DETECTION_MODE: CurationDetectionMode =
+  import.meta.env.VITE_CURATION_DETECTION_MODE === 'client'
+    ? 'client'
+    : 'server';
+
+export const IS_CLIENT_DETECTION_ENABLED =
+  CURATION_DETECTION_MODE === 'client';
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)